code/services/ExtensibleSearchService.php

```diff
@@ -155,4 +155,37 @@ public function getSuggestions($term, $pageID, $limit = 5, $approved = true) {
 		return null;
 	}
 
+
+	/**
+	 *	Retrieve the most relevant search suggestions.
+	 *
+	 *	@parameter <{SEARCH_TERM}> string
+	 *	@parameter <{EXTENSIBLE_SEARCH_PAGE_ID}> integer
+	 *	@return array
+	 */
+
+	public function getTypeahead($request, $pageID) {
```

Comment:
Yeah, assuming it stays as accepting all the variables, I'd rather this variable called `$parameters` or something similar.